Shopify/cli
# Add Tasks component

### WHY are these changes introduced?

We want to be able to convey to the user that a some tasks are running. So far we've been using `listr2`, but we want to follow our own design approach.

~~We'll need to merge [this other PR](https://github.com/Shopify/cli/pull/798) first before we can review this properly.~~

### WHAT is this pull request doing?

This PR introduces a new `renderTasks` method which takes an array of items in the form of: `{title: string, task: Promise}` and displays an animated bar while these tasks are running. At the end it displays a green bar in case of success or a red bar in case any of the tasks has failed to complete.

### How to test your changes?

Run `kitchen-sink` and see the bar for yourself at the end! It should look like this:

![](https://screenshot.click/03-23-s7e6u-4fvbn.gif)

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
